src/modules/content-sqlite/content-sqlite.c

```diff
@@ -846,6 +846,41 @@ static bool synchronous_valid (const char *s)
     return true;
 }
 
+static int process_config (struct content_sqlite *ctx,
```

Comment:
commit message (and PR title): maybe it would be better to say "set sqlite pragmas with flux-config" rather than "config via file" since it need not be an actual file.